composer.json

```diff
@@ -65,10 +65,7 @@
         "preferred-install": {
             "*": "dist"
         },
-        "sort-packages": true,
-        "platform": {
-            "php": "7.4.10"
```

Comment:
I wasn't sure what this was for - Symfony cloud? Should I change to 8.0.x instead of removing?

Reply:
It's here... just kind of as a "best practice" for applications (since this is an app). It won't matter now that we're requiring 8.0, but the idea would that if we are, for example, saying that live-demo works on 8.0, and I accidentally use php 8.1 when working with it, I may download 8.1-only deps to the composer.lock file. Then PHP 8.0 users wouldn't be able to use the library. So yes, even though this won't matter right now, this should stay - but point to 8.0.something.

Reply:
Cool didn't know about this. I've definitely been burned by what you describe.